homeassistant/components/nuheat/climate.py

```diff
@@ -218,9 +228,23 @@ def _set_temperature_and_mode(self, temperature, hvac_mode=None, preset_mode=Non
             target_schedule_mode,
         )
 
-        self._thermostat.set_target_temperature(
-            target_temperature, target_schedule_mode
-        )
+        if target_temperature < self._thermostat.min_temperature:
```

Comment:
We can cap the target temperature with `min` and `max` instead of checking.

Reply:
done.  almost 2am here so I'm off to bed.
g'night!